src/main/docs/guide/httpClient/httpClientImplementations/javanetClient.adoc

```diff
@@ -0,0 +1,15 @@
+To use an implementation based on https://openjdk.org/groups/net/httpclient/intro.html[Java HTTP Client], add the following dependency to your build:
+
+dependency:micronaut-http-client-javanet[]
+
+NOTE: This implementation of the Micronaut HTTP Client is available since Micronaut Framework 4.0.
+
+The implementation based on https://openjdk.org/groups/net/httpclient/intro.html[Java HTTP Client] does not support the following features:
+
+* Proxy Support.
+* Client Filters.
+* Http2.
```

Comment:
We need to add tests, but I think we support Http2 now

Reply:
Ok, remove this once we support it